### PR TITLE
Update dockerfile and create docker compose.yml

### PR DESCRIPTION
**Pull request by jonathanmmm**
Changed Dockerfile, created new docker-compose.yml file and updated Readme, maybe docker-compose installation needed should be mentioned?

Please try it locally if it works on your machines (using linux as host)

Should close this issue:
https://github.com/studo-app/campus-qr/issues/110

### CODE REVIEW

**Comment by jonathanmmm** [comment]
@vonox7 could you or somebody review it?

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+version: '3.9'
+
+services:
+  campus-qr:
+    container_name: campus-qr
+    restart: unless-stopped
+    #not advised to use http in production, using it behind a proxy with https, would be better
```
Comment (vonox7) [inline docker-compose.yml:7]:
```suggestion
    # To run in production, running a reverse-proxy/loadbalancer/gateway with https-termination is required
```

**Review by vonox7 — APPROVED** [review]
Thank you for your collaboration!

File: Dockerfile [hunk 2]
```diff
@@ -1,14 +1,23 @@
-FROM openjdk:8-jre-alpine
+#debian based
+FROM openjdk:11-jdk
 
 ENV APPLICATION_USER ktor
-RUN adduser -D -g '' $APPLICATION_USER
+RUN echo  $APPLICATION_USER
+RUN adduser --disabled-password --gecos '' $APPLICATION_USER
```
[the remark this answers is not in the document]
Comment (jonathanmmm) [reply Dockerfile:6]: @vonox7 its not also more readable, the other version created errors in debian and only worked in alpine images as these were ambigous (multiple options starting with d or g or so)

**Pull request merged**